apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -429,7 +424,15 @@ func pipelineTypes(schema *types.Schemas) *types.Schemas {
 				"rerun": {},
 			}
 		}).
-		MustImport(&Version, v3.SourceCodeRepository{}).
-		MustImport(&Version, v3.PipelineExecutionLog{})
+		MustImport(&Version, v3.PipelineExecutionLog{}).
+		MustImportAndCustomize(&Version, v3.SourceCodeCredential{}, func(schema *types.Schema) {
```

Comment:
if is is not allowed to be created via API, we should probably remove the POST method from collectionMethods.

Reply:
Sure, updated